src/en/space-station-14/round-flow/proposals/revolutionaries-codeword-rework.md

```diff
@@ -0,0 +1,271 @@
+
+Methods of deconversion include:
+  - Harm baton, heals **10** LHP per hit
+  - Being in cuffs heals **0.5** LHP / second
+  - Wearing an electro pack heals **0.5** LHP / second
+
+Loyal crew are also able to passively heal LHP at **0.1** LHP / second, though as this does not happen for Revolutionaries it can not be used for deconversion. This passive heal is disabled for **10** seconds upon taking LHP damage.
+
+A deconverted Revolutionary can be reconverted again. If a Headrev is deconverted (>= 0 LHP) they lose their antag status, and being converted to the Revolution makes you a normal Revolutionary. 
+
+### Death & LHP Damage
+
+Authoritarian regimes killing of protesting civilians causing further unrest, or freedom fighters getting a shock when they first murder someone, are concepts that are closely tied to a revolution. They are also ideas that would be suitable mechanically to combat murder sprees from either side during a Revolution round. The following section is only relevant for crew and Revs; non-crew characters (pets, mobs) and other antags (traitors, Nukies) do not interact with the following mechanic.
+
+If a Revolutionary *or crewmember* dies by the hand of another crewmember (being dealt the damage that put them in a critical state), they will deal **15** non-split LHP in a **8** tile line-of-sight radius around their body. While remaining dead they will also provide a passive **-0.1** LHP / second in a **4** tile line-of-sight radius. This works in the same way but in reverse if a Revolutionary performs the kill, healing LHP upon death when killed by a Revolutionary/Crewmember. The initial burst also has a **120** second cooldown, to avoid repeatedly reviving and killing the same person twice to trigger the effect. This means both sides are discouraged from indiscriminately killing, favoring limited killing and kidnapping. 
+
+If the death of a *Revolutionary* is at the hand of a *crewmember*, the initial burst of LHP damage is able to convert. This is not true for the death of a crewmember by the hand of a crewmember (otherwise large-scale death in other gamemodes could risk spawning Revolutionaries without any headrev). 
+
+Being converted/deconverted by a character dying displays a special conversion message, to indicate it was the reason for being converted/deconverted.
+
+There is an interesting dichotomy that can form here where Revolutionaries have an easier time to "heal" revolutionary fervor by dealing LHP damage to each other via chanting codewords. On the other hand, Security has more access to stun weaponry and as such have an easier time subduing Revolutionary groups non-lethally. 
```

Comment:
Or Sec can just kill everyone and let the Admins sort it out.

Reply:
See: Death & LHP Damage